src/__test__/ProductList.test.tsx

```
@@ -1,45 +1,28 @@
 
-vi.mock('@/shared/components/Toast', () => ({
-  Toast: {
-    show: vi.fn(),
-  },
-}));
+export const renderProductListPage = () =>
+  render(
+    <DataProvider>
+      <ProductListPage />
+    </DataProvider>
+  );
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)